tests/integration/resources/test_logs_indexes.py

```diff
@@ -3,16 +3,54 @@
 # This product includes software developed at Datadog (https://www.datadoghq.com/).
 # Copyright 2019 Datadog, Inc.
 
-import pytest
+import os
+import json
+import logging
+import urllib.request
 
-from tests.integration.helpers import BaseResourcesTestClass
 from datadog_sync.models import LogsIndexes
+from datadog_sync.cli import cli
+from datadog_sync.utils.resource_utils import open_resources
+from tests.integration.helpers import BaseResourcesTestClass, save_source_resources
 
 
 class TestLogsIndexesResources(BaseResourcesTestClass):
     resource_type = LogsIndexes.resource_type
     field_to_update = "filter.query"
 
-    @pytest.mark.skip(reason="delete is not supported for logs-indexes")
-    def test_resource_cleanup(self):
-        pass
+    def test_resource_cleanup(self, runner, caplog):
+        caplog.set_level(logging.DEBUG)
+
+        source_resources, _ = open_resources(self.resource_type)
+
+        # Remove the first resource from the source
+        first_index = list(source_resources.keys())[0]
```

Comment:
Where does the data for this test come from? It'd be easier to visually verify that it's correct if it was hardcoded in the test itself. I think the test trivially passes if `source_resources` is length 1, or if all of its elements are identical, and it's not clear whether that's the case here.